.github/workflows/pre-deploy.yaml

```diff
@@ -40,3 +40,21 @@ jobs:
         run: yarn --frozen-lockfile
       - name: Run build
         run: yarn build
+
+  audit:
+    name: Security Audit
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: yarn --frozen-lockfile
+      - name: Run Audit
+        run: yarn sec-audit --fail-on 'critical'
```

Comment:
@hspotlight Integrated with Security Audit. I think it ready to merge.
However, I set it to fail on 'critical' level since there are still outdated dependencies that marked as high.

Reply:
thank you, I gonna to merge now. 😄